test/hotspot/jtreg/serviceability/jvmti/GetOwnedMonitorInfo/GetOwnedMonitorInfoTest.java

```diff
@@ -86,6 +86,7 @@ public static void runTest(boolean isVirtual, boolean jni) throws Exception {
                 System.out.println("Thread doing JNI call: "
                                    + Thread.currentThread().getName());
 
+                Thread.yield();
```

Comment:
Please add a comment explaining why the yield is present

Reply:
Thanks, David. Added a comment. Please, let me know if new comment is not good enough. I do not want to explain the whole scenario there.

Reply:
The comment is sufficient - thanks

Reply:
Thanks!